app/services/by_month_project_fetcher_service.rb

```diff
@@ -27,6 +27,19 @@ def confirmed_openers_by_team(month, year, team)
     sort_by_conditions_met_and_name(projects)
   end
 
+  def revised_openers_by_team(month, year, team)
+    users = User.by_team(team)
+
+    projects = Conversion::Project.assigned_to_users(users).conversion_date_revised_from(month, year).includes(:tasks_data)
```

Comment:
I've hit the bullet issue you had straight away:

`ActiveRecord::EagerLoadPolymorphicError in Team::Opening::ProjectsController#confirmed`

It feels like the issue is that `in_batches` cannot cope with our super new use of delegated types? The existing tests use doubles, yours uses ActiveRecord objects, which might explain the different results?

Given how expensive this view is already, I am not worried about eager loading the tasks_data here and would be happy to exlude it via the Bullet config - we can have a chat about this if you like, it's pretty late in the day and I might be missing something!